pkg/api/v1/defaults.go

```diff
@@ -389,3 +389,32 @@ func SetDefaults_HostPathVolumeSource(obj *v1.HostPathVolumeSource) {
 		obj.Type = &typeVol
 	}
 }
+
+func SetDefaults_SecurityContext(sc *v1.SecurityContext) {
+	pt := true
```

Comment:
For consistency, it should have name `ape` (as in the fuzzer).

Reply:
can do, I was debating over what naming I liked better, honestly I just wish `&true` and `&false` worked